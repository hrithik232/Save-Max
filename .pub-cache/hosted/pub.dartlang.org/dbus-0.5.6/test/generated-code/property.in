<node>
  <interface name="com.example.Test">
    <property name="Value" type="s"/>
  </interface>
</node>
