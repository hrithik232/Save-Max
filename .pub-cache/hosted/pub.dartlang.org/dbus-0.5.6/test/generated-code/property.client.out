import 'package:dbus/dbus.dart';

class ComExampleTest extends DBusRemoteObject {
  ComExampleTest(DBusClient client, String destination, DBusObjectPath path) : super(client, destination, path);

  /// Gets com.example.Test.Value
  Future<String> getValue() async {
    var value = await getProperty('com.example.Test', 'Value', signature: DBusSignature('s'));
    return (value as DBusString).value;
  }

  /// Sets com.example.Test.Value
  Future<void> setValue (String value) async {
    await setProperty('com.example.Test', 'Value', DBusString(value));
  }
}
